### DISCUSSION TITLE
Missing Syphid6 pendind installation

### DISCUSSION THREAD

**Discussion by Stephan35** [bpozdena/OneDriveGUI#24] [Q&A] [answered]
Hi,
I would like to try GUI, but installation failed , and i can not install Syphid6 (and neither Syphid2)
I tried to compile it, but not easy way for me.
Thanks for any help.

**Comment by bpozdena** [comment]
The name of the pip package is 'pyside6' not 'Syphid6' .

Can you share the commands you are executing and the output you receive from them?

**Reply by Stephan35** [reply]
Hello,
Yes, sorry for this mistake, missing PySide6 ==6.2.2.1

![image](https://user-images.githubusercontent.com/16188172/154841560-0bc33872-7350-4aad-8600-c86d20b35385.png)

**Reply by GalaxyCurse** [reply]
Do you have the latest version of pip3 installed?

**Reply by Stephan35** [reply]
> 
it is pip 18.1 of python 3.7

![image](https://user-images.githubusercontent.com/16188172/154848544-8b651c5d-8fba-4b80-8e13-c0c4f4e4a435.png)

**Comment by bpozdena** [comment, marked as answer]
Not really sure what the issue is. Each distro behaves a bit differently especially if Python2 was the default version. I can think of the bellow potential solutions:

A) Try specifying python3:
```
python3 -m pip install -r requirements.txt
```
B) Try to upgrade pip, then try again.
```
python3 -m ensurepip --upgrade
``` 

C) You can also try to install the dependencies manually. The GUI should theoretically work with any PySide6 version:
```
pip3 install pyside6
pip3 install requests
```
